imod_coupler/drivers/dfm_metamod/dfm_metamod.py

```diff
@@ -160,14 +161,19 @@ def update(self) -> None:
 
         # sub timestepping between metaswap and dflow
         subtimestep_endtime = t_begin
-        for _ in range(self.number_dflowsteps_per_modflowstep):
-            subtimestep_endtime += self.delt / self.number_dflowsteps_per_modflowstep
+        for idtsw in range(self.number_substeps_per_modflowstep):
+            subtimestep_endtime += self.dtsw
+            self.msw.start_surface_water_time_step(idtsw)
 
             while (
                 self.dfm.get_current_time()
                 < days_to_seconds(subtimestep_endtime) - self.time_eps
             ):
                 self.dfm.update()
```

Comment:
we tell dfm to update until iuts current time is not smaller than subtimetstep_endtime. Let's say that after the update is is indeed not smaller, but a lot larger. would that give problems? can  we prevent that?

Reply:
good question, I don't really know. I assume the delt of dflow is small enough to not give problems, but I'm not sure. But I did not change this in this branch so we should make a separate task out of it to figure this out.

Reply:
I'l put an issue on the backlog for that